modules/test/playwright/tests/client-extension-web/pages/ClientExtensionsPage.ts

```diff
@@ -36,6 +54,36 @@ export class ClientExtensionsPage {
 		await this.editMenuItem.click();
 	}
 
+	getRowByText(text: string) {
+		return this.page
+			.locator('.dnd-tbody')
+			.locator('.dnd-tr')
+			.filter({
+				has: this.page.getByText(text, {exact: true}).first(),
+			});
+	}
```

Comment:
We already have very similar code in two places in `frontend-data-set-views`, [here](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-views-web/pages/data_set/tabs/FiltersPage.ts#L280-L287) and [here](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-views-web/pages/data_set/tabs/VisualizationModesPage.ts#L101-L105). Would it make sense to make it a global util? The rule of three did kick in, although you could say it's two, with two modules.

Not a blocker for task.

Reply:
yep that makes sense. And we need it because when FDS gets migrated to clay, this specific thing will miserably fail and so we need to switch the piece easily